core/LSU.hpp

```diff
@@ -164,8 +167,23 @@ namespace olympia
         const int complete_stage_;
 
         // Load/Store Pipeline
-        using LoadStorePipeline = sparta::Pipeline<LoadStoreInstInfoPtr>;
-        LoadStorePipeline ldst_pipeline_;
+        const uint32_t ldst_pipeline_count_;
+        uint32_t ldst_active_pipeline_idx = 0;
+
+        struct LSPayload
+        {
+            LoadStoreInstInfoPtr ldst_inst_info_ptr;
+            uint32_t ldst_pipeline_idx;
```

Comment:
Pls initialize or make `const`:
```suggestion
            uint32_t ldst_pipeline_idx = std::numeric_limits<uint32_t>::max();
```